types/src/on_chain_config/aptos_features.rs

```diff
@@ -274,6 +279,29 @@ impl Features {
     pub fn is_refundable_bytes_enabled(&self) -> bool {
         self.is_enabled(FeatureFlag::REFUNDABLE_BYTES)
     }
+
+    pub fn get_max_identifier_size(&self) -> u64 {
+        if self.is_enabled(FeatureFlag::LIMIT_MAX_IDENTIFIER_LENGTH) {
+            IDENTIFIER_SIZE_MAX
+        } else {
+            LEGACY_IDENTIFIER_SIZE_MAX
+        }
+    }
+
+    pub fn get_max_binary_format_version(&self, gas_feature_version_opt: Option<u64>) -> u32 {
+        // For historical reasons, we support still < gas version 5, but if a new caller don't specify
```

Comment:
the comment seems to indicate we support gas version < 5, but it's overridden to 5. I guess it means we allow specifying <5? needs to be clearer.

nits:
still support
new caller doesn't specify

why does it matter when it was introduced? is there anything more special about 5 for why it's used as a default? (that could be useful to comment)

Reply:
Will try to make sense of comment haha, I have just copied the whole thing and made it a `Feature` member function

Reply:
Updated: removed gas feature version and completely and switched to 5 as default. I think it has been long enough so we should be good wirh just using feature flags. cc @vgao1996, does this look good?

Reply:
As with any feature-gating removal, we should just run replay verify and see if anything breaks.

Reply:
all passed

Reply:
Then it should be good